### ISSUE TITLE
Standalone TSCH

### ISSUE THREAD

**Issue by nikooiko** [contiki-ng/contiki-ng#955]
Similar to [Standalone RPL-lite](https://github.com/contiki-ng/contiki-ng/issues/765), also quite interesting is the idea of doing the same for the TSCH and 6TiSCH, that work alongside RPL.

Is this doable?
Is there any reason for not doing so?
@simonduq what's your opinion about this?

**Comment by simonduq** [comment]
Yes we'd love to make TSCH/6TiSCH also standalone. This is a little more challenging than for RPL-Lite though, because of how tightly coupled it is to hardware abstraction layers and timers etc. But yes, definitely doable, definitely desirable. Only reason for not doing so is lack of manpower :) I'll be happy to support and review if someone takes this forward. Just like for RPL-Lite.
